### DISCUSSION TITLE
Problem!

### DISCUSSION THREAD

**Discussion by xKyKaxz** [w-okada/voice-changer#1321] [Q&A]
I decided to download a voice changer to play a joke on my friends, I did everything as per the instructions, I went into discord and I can't hear what I'm saying into my microphone and the virtual cable doesn't transmit sound, what should I do?

**Comment by AnnoyingFox** [comment]
> I decided to download a voice changer to play a joke on my friends, I did everything as per the instructions, I went into discord and I can't hear what I'm saying into my microphone and the virtual cable doesn't transmit sound, what should I do?

If you want to hear what you are speaking then you need to set your speakers or headset on the monitor field.
Also, uninstall vb cable and install virtual audio cable instead. Vb cable doesn't work with discord.

**Reply by xKyKaxz** [reply]
Thank
